server/sonar-server/src/main/resources/org/sonar/server/computation/ws/quality_gate_status-example.json

```diff
@@ -0,0 +1,70 @@
+{
+  "qualityGateDetails": {
+    "level": "ERROR",
+    "conditions": [
+      {
+        "metric": "new_coverage",
+        "op": "LT",
+        "period": "1",
+        "warning": "",
```

Comment:
the format in datastore is internal and does not follow the same constraints as WS.